data/ecosystems/n/noir-lang.toml

```diff
@@ -143,3 +143,225 @@ url = "https://github.com/vezenovm/simple_shield"
 
 [[repo]]
 url = "https://github.com/whitenois3/nplate"
+
+[[repo]]
+url = "https://github.com/BattleZips/BattleZips-Noir"
+
+[[repo]]
+url = "https://github.com/colinnielsen/ecrecover-noir"
+
+[[repo]]
+url = "https://github.com/Ethan-000/halo2_backend"
+
+[[repo]]
+url = "https://github.com/Miyamura80/ZKMicrophone"
+
+[[repo]]
+url = "https://github.com/resurgencelabs/types"
+
+[[repo]]
+url = "https://github.com/SetProtocol/noir-rsa"
+
+[[repo]]
+url = "https://github.com/tonk-gg/dappicom"
+
+[[repo]]
+url = "https://www.noirguardians.io/"
+
+[[repo]]
+url = "https://github.com/thor314/circuit-examples"
+
+[[repo]]
+url = "https://github.com/Kryha/scaffold-eth-2-noir"
+
+[[repo]]
+url = "https://github.com/scaffold-eth/scaffold-eth-2"
```

Comment:
I think this one can be removed since it's primarily related to the Ethereum ecosystem